### PR TITLE
Low level doc improvements

### PR DESCRIPTION
**Pull request by jessiihu**

### CODE REVIEW

File: mint.json [hunk 1]
```diff
@@ -54,10 +54,22 @@
       "pages": ["introduction", "quickstart", "development"]
     },
     {
-      "group": "Settings",
+      "group": "Customizing your Docs",
```
Comment (handotdev) [inline mint.json]: Can we switch the title to just `Customizations`?

File: mint.json [hunk 2]
```diff
@@ -84,12 +96,12 @@
...
+      "group": "API Reference",
       "pages": [
-        "api-playground/overview",
-        "api-playground/configuration",
-        "api-playground/authentication",
-        "api-playground/openapi"
+        "api-reference/overview",
+        "api-reference/configuration",
+        "api-reference/authentication",
+        "api-reference/openapi"
```
Comment (handotdev) [inline mint.json]: I would recommend API Playground over API Reference as `API Reference` is widely used to refer to the API endpoints. Something like this for instance https://www.flipt.io/docs/reference/overview

File: settings/page-structure.mdx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Navigation"
+title: "Page Structure"
```
Comment (handotdev) [inline settings/page-structure.mdx]: When I hear Page Structure, I think more of how an individual page is set up, whereas this one is more so about the order of the pages.

**Review by hanywang2 — COMMENTED** [review]
Great job Jessi! Just left a few comments for suggestions

**Pull request merged**